integration/testrunner/grpcrunner/grpc.go

```diff
@@ -87,15 +125,17 @@ func TestGRPCEndpoint(ctx context.Context, tb testing.TB, g *GRPC) {
 		g.BigQueryClient = bqClient
 	}
 
-	g.runHelloCheck(ctx, tb)
-	g.runFailCheck(ctx, tb)
-	g.runFibonacciCheck(ctx, tb)
-	g.runAdditionCheck(ctx, tb)
-	g.runFailOnFourCheck(ctx, tb)
+	justificationRequired := strings.Contains(g.EndpointURL, "go")
+
+	g.runHelloCheck(ctx, tb, justificationRequired)
+	g.runFailCheck(ctx, tb, justificationRequired)
+	g.runFibonacciCheck(ctx, tb, justificationRequired)
+	g.runAdditionCheck(ctx, tb, justificationRequired)
+	g.runFailOnFourCheck(ctx, tb, justificationRequired)
```

Comment:
Instead of passing this flag all over the places, I recommend adding it as a flag in the GRPC struct https://github.com/abcxyz/lumberjack/blob/main/integration/testrunner/grpcrunner/grpc.go#L38

Reply:
it doesn't make sense there, as this is specific to the test, not the test suite.

Reply:
I don't understand. Isn't this a flag specifying whether to check justification in the audit logs? The GRPC struct provides the necessary configuration to run the test such as URL. Which part doesn't fit?

Reply:
Yes, and we may not want to check for justification in all logs. Like right here, we don't want to check justification on Java tests.

Reply:
Right, in the GRPC structs for java, we'd set this flag to false.

Reply:
got it, changed.